apps/test/unit/lib/levelbuilder/lesson-editor/SearchBoxTest.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {mount, shallow} from 'enzyme';
+import sinon from 'sinon';
+import {expect} from '../../../../util/reconfiguredChai';
+import SearchBox from '@cdo/apps/lib/levelbuilder/lesson-editor/SearchBox';
+
+describe('SearchBox', () => {
+  let defaultProps, constructOptions, fetchSpy;
+
+  beforeEach(() => {
+    constructOptions = sinon.spy();
+    fetchSpy = sinon.stub(window, 'fetch');
```

Comment:
instead of stubbing `fetch`, would `sinon.fakeServer` solve your problem?

Reply:
I tried `sinon.fakeServer` first but it looks like that doesn't work because SearchBox uses `fetch`. Brad wrote up a [detailed explanation](https://github.com/code-dot-org/code-dot-org/pull/34522/files#r418255345) a few months ago. His solution there was also to stub `fetch`.

Reply:
Makes sense! Thanks for pursuing this. I noticed in Brad's write-up that there is a separate library that could be used to stub fetch, but I don't think it's necessary for us to embark on that now.